addons/mail/static/src/core/web/messaging_menu.js

```diff
@@ -71,6 +74,14 @@ export class MessagingMenu extends Component {
         return tab === "chat" ? ["chat", "group"] : [tab];
     }
 
+    get canBeInstalled() {
+        return (
+            this.installPrompt.canPromptToInstall &&
+            !isDisplayStandalone() &&
+            browser.location.pathname !== "/web/tests"
+        );
+    }
```

Comment:
Could this be moved to `installPrompt` service?
The `isDisplayStandalone()` and `browser.location.pathname !== "/web/tests"` does not seem to be conceptually specific to messaging menu

Reply:
No problem, I will move it to the service directly